airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -58,7 +59,9 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
               .withCatalog(run.getOutputCatalog())
               .withResourceRequirements(syncInput.getDestinationResourceRequirements());
 
-          normalizationActivity.normalize(jobRunConfig, destinationLauncherConfig, normalizationInput);
+          final StandardNormalizationSummary normalizationSummary =
+              normalizationActivity.normalize(jobRunConfig, destinationLauncherConfig, normalizationInput);
+          persistActivity.persist(connectionId, run.withStandardNormalizationSummary(normalizationSummary));
```

Comment:
Looking at the [ConnectionManagerWorkflowImpl](https://github.com/airbytehq/airbyte/blob/master/airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java), you can see that the `StandardSyncOutput` object that is returned by this run method is also [returned by the method](https://github.com/airbytehq/airbyte/blob/ead33566776befc41d79b2f193895328e06f807b/airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java#L555) that calls this `run()`, which in turn is passed to either `reportFailure` or `reportSuccess`, both of which pass it to the `JobCreationAndStatusUpdateActivity`. This is where the standard sync output is written to the db with [jobPersistence.writeOutput()](https://github.com/airbytehq/airbyte/blob/ead33566776befc41d79b2f193895328e06f807b/airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/JobCreationAndStatusUpdateActivityImpl.java#L156).

All of the above is the context for the main point I am trying to make here, which is that we need to make sure that the `StandardSyncOutput` object returned by this `run()` method includes the `normalizationSummary` if we ran normalization. I think calling `run.withNormalizationSummary()` does actually update the object as well, but to be extra clear that this update is happening to the object, I would prefer that we reassign `run` to the output of `run.withNormalizationSummary()` (which means we will also need to change that variable to not be `final`, as that prevents reassignment).

I am also not a huge fan of the variable name `run` - I would prefer if it was named something clearer like `output` or `standardSyncOutput`